jupyter_ydoc/ydoc.py

```diff
@@ -132,6 +272,15 @@ def get_cell(self, index: int) -> Dict[str, Any]:
         return cell
 
     def append_cell(self, value: Dict[str, Any], txn=None) -> None:
+        """
+        Adds a cell to `self._ycells`.
+
+        :param value: A cell.
+        :type value: Dict[str, Any]
+
+        :param txn: A YTransaction, defaults to None
+        :type txn: :class:`YTransaction`, optional.
```

Comment:
Sometimes you use ":class:`YTransaction` | None", it should be consistent.

Reply:
Where do I use ":class:YTransaction | None" ?

Reply:
I mean `| None` in general, instead of `Optional[...]`. The former is for Python 3.10+, so we should still use the later.